src/helperFunctions/hash.py

```diff
@@ -45,5 +52,19 @@ def check_similarity_of_sets(pair_of_sets, all_sets):
     return True
 
 
+def _is_elf_file(file_object):
+    file_type = file_object.processed_analysis['file_type']['full'].lower()
+    return re.search(r'elf', file_type) is not None
+
+def get_imphash(file_object):
+    imphash = None
+    if _is_elf_file(file_object):
+        try:
+            elf = lief.parse(file_object.file_path)
+            imphash = md5(','.join(elf.imported_functions).encode()).hexdigest()
+        except Exception as e:
+            logging.error('Could not compute imphash for ELF {}: {}'.format(file_object.file_path, str(e)))
```

Comment:
e does not provide the error class. You can use something like this to get the full error 
`'Could not compute imphash for ELF {}: {} {}'.format(file_object.file_path, sys.exc_info()[0].__name__, e)`

Reply:
Aaaah, nein nicht so.
Das `sys.exc_info()[0].__name__` Konstrukt sollte aus der Code-Basis entfernt werden. Den Fehlertyp erhält man einfacher mit type(e), dann kann man sich auch den Import von `sys` sparen.